edge-modules/api-proxy-module/Cargo.toml

```diff
@@ -0,0 +1,50 @@
+[package]
+name = "api-proxy-module"
+version = "0.1.0"
+authors = ["Azure IoT Edge Devs"]
+license = "MIT"
+edition = "2018"
+
+[dependencies]
+base64 = "0.11"
+bytes = "0.5"
+futures-core = "0.3"
+futures-channel = "0.3"
+futures-util = "0.3"
+hmac = "0.7"
+http = "0.2"
+hyper = { version = "0.13", features = ["stream"] }
+hyper-uds = { path = "./rust-sdk/hyper-uds" }
+lazy_static = "1.2"
+log = "0.4"
+mqtt3 = { path = "../../mqtt/mqtt3" }
+native-tls = "0.2"
+percent-encoding = "2"
+regex = "1.3.9"
+serde = { version = "1", features = ["derive", "rc"] }
+serde_derive = "1"
+serde_json = "1"
+sha2 = "0.8"
+tokio = { version = "0.2.22", features = ["tcp", "time", "process"] }
+tokio-io-timeout = "0.4"
+tokio-tls = "0.3"
+tungstenite = "0.10"
+url = "2"
+anyhow = "1.0.32"
+reqwest = { version = "0.10.7", features = ["blocking", "json"] }
+chrono = "0.4.13"
+md5 = "0.7.0"
+azure-iot-mqtt = { path = "./rust-sdk/azure-iot-mqtt" }
+structopt = "0.3"
+env_logger = "0.7"
+futures = "0.3.5"
+envsubst = "0.2.0"
+edgelet-client = { path = "../../mqtt/edgelet-client" }
```

Comment:
@dmolokanov, do you think this is fine or we want to publish edgelet-client as a crate?

Reply:
A crate in the long run would be better. If it is just us using it it is ok but I suspect we are not going to be the only one using it.

Reply:
Well, it depends. If at some point in time we decide, we want to make an official rust SDK for edge, we publish it somewhere. If we decide we want to tie up edgelet client and edgelet API, it's better we publish it somewhere. Until that it is easier for everyone to link it in the repo.